actions/setup-argo/README.md

```diff
@@ -1,5 +1,7 @@
 # Setup Argo
 
+> If you are at Grafana Labs, follow these steps in the [internal documentation](https://enghub.grafana-ops.net/docs/default/component/deployment-tools/platform/continuous-delivery/argo-workflows/) first.
```

Comment:
I'd drop this one. This just get the `argo` CLI installed. `trigger-argo-workflows` is the one that is more Grafana Labs specific.